bootstrap.py

```diff
@@ -639,6 +639,26 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = "katello-capsule"
```

Comment:
Given that the upstream packages will be installed downstream too, I don't think we have to explicitly check for the downstream names, no?